charts/headlamp/templates/deployment.yaml

```diff
@@ -37,7 +37,9 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - "-in-cluster"
-            - "-plugins-dir=/headlamp/plugins"
+            {{- with .Values.config.pluginDir}}
```

Comment:
As the flag is named `plugins-dir`, I think it would make sense the variable to be `pluginsDir`. WDYT?